ci/ray_ci/tester.py

```diff
@@ -174,7 +173,10 @@ def _get_test_targets(
     )
     flaky_tests = _get_flaky_test_targets(team, yaml_dir)
 
-    return [test for test in test_targets if test and test not in flaky_tests]
+    if get_flaky_tests:
+        return [test for test in test_targets if test and test in flaky_tests]
+    else:
+        return [test for test in test_targets if test and test not in flaky_tests]
```

Comment:
maybe use a for loop rather than a comprehension? of maybe use a `filter()` function with a lambda?

the two comprehensions look very similar right now.